libs/feature/teaching/src/lib/author/authors-form.tsx

```diff
@@ -38,15 +41,15 @@ export function AuthorsForm({ subtitle, emptyString }: { subtitle: string; empty
 
 	return (
 		<Form.SidebarSection>
-			<Form.SidebarSectionTitle title="Autoren" subtitle={subtitle} />
+			<Form.SidebarSectionTitle title="Autoren" subtitle={subtitle}/>
 
 			<IconButton
 				type="button"
 				data-testid="author-add"
 				onClick={() => setOpenAddDialog(true)}
 				title="Hinzufügen"
 				text="Hinzufügen"
-				icon={<PlusIcon className="h-5" />}
+				icon={<PlusIcon className="h-5"/>}
```

Comment:
linting eingeschaltet?

Reply:
Ja.

Reply:
in den vscode einstellungen die im repo liegen, ist das automatisch beim dateispeichern. ich überprüfe das mal